desktop/components/marketing_signup_modal/index.coffee

```diff
@@ -54,7 +54,8 @@ module.exports = class MarketingSignupModal extends Backbone.View
 
     # This is just for the Seattle Art Fair campaign and should be removed after
     # the fair closes.
-    slug = 'ca3' if !slug and sd.CURRENT_PATH is '/seattle-art-fair-2017'
```

Comment:
Not empty string b/c that would match against the homepage, but it could be `TARGET_CAMPAIGN_URL=null` because we [`JSON.parse`](https://github.com/artsy/force/blob/master/desktop/config.coffee#L114) config variables.